src/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 :nth-child(-n + 0) {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url(../src/images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url(../src/images/star-active.svg);
+}
```

Comment:
Checklist:
2. [CODE STYLE] - If several selectors MUST always have the same styles, group them using , to prevent accidental out ot sync in future
![image](https://github.com/mate-academy/layout_stars/assets/63243136/67f87b7f-dfb6-4f09-852b-d90674d63372)